examples/testapp/src/components/SDKConfig/SDKConfig.tsx

```diff
@@ -14,14 +14,14 @@ import { Preference } from '@coinbase/wallet-sdk/dist/core/provider/interface';
 import React, { useCallback, useMemo, useState } from 'react';
 import { keccak256, slice, toHex } from 'viem';
 
-import { useCBWSDK } from '../../context/CBWSDKReactContextProvider';
+import { useConfigParams } from '../../context/ConfigParamsContextProvider';
 
-function computeDataSuffix(value: string): string {
+function computeDataSuffix(value: string): `0x${string}` {
   return slice(keccak256(toHex(value)), 0, 16);
 }
 
 export function SDKConfig() {
-  const { config, setConfig } = useCBWSDK();
+  const { config, setConfig } = useConfigParams();
```

Comment:
nit: could this just be `useConfig` and the provider just be `ConfigProvider`?

Reply:
updated!